stories/imdb/stories/checklist.js

```diff
@@ -108,10 +115,11 @@ export default () => (
               tree={tree}
               path={['root', 'criteria']}
               fields={schemas.movies.fields}
-              typeComponents={TypeMap}
-              mapNodeToProps={field =>
-                field.key === 'searchNumber' ? { showBestRange: true } : {}
-              }
+              mapNodeToProps={F.mergeOverAll([
```

Comment:
Now that this is common, we might make `mapNodeToProps` support an array of functions and take care of the `mergeOverAll` call. Not now though, because the goal of killing `typeComponents` was to reduce the amount of magic and surface area to keep the API as simple as possible to understand. We might never do it - but I had that thought and didn't want it to only live in my head 😄